src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -196,10 +209,15 @@ public void verifyIsEmailOfInstructorOfCourse(String instructorEmail, String cou
     public InstructorAttributes updateInstructorByGoogleIdCascade(
             InstructorAttributes.UpdateOptionsWithGoogleId updateOptions)
             throws InvalidParametersException, EntityDoesNotExistException {
+
         InstructorAttributes originalInstructor =
                 instructorsDb.getInstructorForGoogleId(updateOptions.getCourseId(), updateOptions.getGoogleId());
         InstructorAttributes updatedInstructor = instructorsDb.updateInstructorByGoogleId(updateOptions);
 
+        boolean isOriginalInstructorDisplayed = originalInstructor.isDisplayedToStudents();
+        verifyAtLeastOneInstructorIsDisplayed(originalInstructor.courseId, isOriginalInstructorDisplayed,
```

Comment:
This check is meaningless now that `instructorsDb.updateInstructorByGoogleId` has been called before.

Reply:
`UpdateOptionsWithGoogleId` has courseId and googleId as members, should i add the isDisplayed field and write a getter method, in order to retrieve the new value of `isDisplayedToStudents` and move the check `verifyAtLeastOneInstructorIsDisplayed`  before the update is made

Reply:
Another place to make the `verifyAtLeastOneInstructorIsDisplayed` is in `instructorsDb` when 
`instructorsDb.updateInstructorByGoogleId` is called but I dont think that it would be the correct design

Reply:
Precondition check can be done in the same way as `FeedbackSessionsLogic#updateFeedbackSession`.

Reply:
Thanks for the pointer. 👍